portal-impl/src/com/liferay/portal/servlet/filters/language/LanguageFilter.java

```diff
@@ -118,6 +97,22 @@ protected void processFilter(
 	protected String translateResponse(
 		HttpServletRequest httpServletRequest, String content) {
 
+		Enumeration<String> parameterNamesEnumeration =
+			httpServletRequest.getParameterNames();
+
+		PortletConfig portletConfig = null;
+
+		while (parameterNamesEnumeration.hasMoreElements()) {
+			String parameterName = parameterNamesEnumeration.nextElement();
+
+			if (parameterName.contains("_INSTANCE_")) {
+				portletConfig = PortletConfigFactoryUtil.get(
+					PortletIdCodec.decodePortletName(parameterName));
+			}
+		}
+
```

Comment:
@dantewang Could you please explain the logic here? There are two types of portlet, one can have multiple instances, the other can not. It could be that you are not an instancable portlet, which does not have "_INSTANCE_".

Also, please add test to cover this change